js/package.json

```diff
@@ -23,6 +23,7 @@
     "express-session": "~1.15.6",
     "express-ws": "~4.0.0",
     "glob": "~7.1.3",
+	"readline-sync": "^1.4.9",
```

Comment:
This needs to start with "~" instead of "^", since we want our versions to vary by no more than a patch version to maintain consistency over time.